src/cases/components/roles-and-access/roles-and-access.component.ts

```diff
@@ -55,6 +59,10 @@ export class RolesAndAccessComponent implements OnInit, OnChanges {
     }
   }
 
+  public removeCashedCase(): void {
+    this.caseNotifier.cachedCaseView = null;
```

Comment:
Minor change - please introduce a function removeCachedCase in CaseNotifier instead if marking it to null